.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
```

Comment:
`code.google.com` is legacy. just remove it and use `golang.org/x`.
Though really this entire line could be removed as it's already pulled in.